src/sagemaker_xgboost_container/training.py

```diff
@@ -37,11 +37,21 @@ def run_algorithm_mode():
         'SM_HOSTS'
         'SM_CURRENT_HOST'
         'SM_MODEL_DIR'
+        'SM_CHECKPOINT_CONFIG_FILE'
 
     """
     # TODO: replace with CSDK constants in sagemaker_containers._env
-    train_config = json.load(open(os.getenv(sm_env_constants.SM_INPUT_TRAINING_CONFIG_FILE), "r"))
-    data_config = json.load(open(os.getenv(sm_env_constants.SM_INPUT_DATA_CONFIG_FILE), "r"))
+    with open(os.getenv(sm_env_constants.SM_INPUT_TRAINING_CONFIG_FILE), "r") as f:
+        train_config = json.load(f)
+    with open(os.getenv(sm_env_constants.SM_INPUT_DATA_CONFIG_FILE), "r") as f:
+        data_config = json.load(f)
+
+    checkpoint_config_file = os.getenv(sm_env_constants.SM_CHECKPOINT_CONFIG_FILE)
```

Comment:
Very nitpick, but I would just follow the pattern established above and skip the variable initialization for checkpoint_config_file.